packages/core/src/Stores/ui-store.js

```diff
@@ -550,8 +550,9 @@ export default class UIStore extends BaseStore {
     }
 
     @action.bound
-    toggleAccountTransferLimitModal(state_change = !this.is_account_transfer_limit_modal_visible) {
+    toggleAccountTransferLimitModal(state_change = !this.is_account_transfer_limit_modal_visible, response) {
         this.is_account_transfer_limit_modal_visible = state_change;
+        this.message = response.message;
```

Comment:
Probably better to rename this to `account_transfer_limit_message` .
```suggestion
        this.account_transfer_limit_message = message;
```